advanced/keyboard-shortcuts.rst

```diff
@@ -104,34 +126,36 @@ How
-   * press ``cmd`` + ``i`` to set the text in *italics*.
+   * press :kbd:`ctrl` :kbd:`i` to set the text in *italics*.
 
 Key Combinations
-   ==============================  =============================================
-   Key / key combination           Function
-   ==============================  =============================================
-   ``ctrl`` + ``u``                Format text underlined
-   ``ctrl`` + ``b``                Format text in **bold**
-   ``ctrl`` + ``i``                Format text in *italics*
-   ``ctrl`` + ``s``                Format text as  ̶s̶t̶r̶i̶k̶e̶t̶h̶r̶o̶u̶g̶h̶
-   ``ctrl`` + ``v``                Paste text from clipboard
-   ``ctrl`` + ``shift`` + ``v``    Paste text from clipboard (as plain text)
-   ``shift`` + ``ctrl`` + ``f``    Remove formatting of text
-   ``shift`` + ``ctrl`` + ``y``    Remove formatting of the whole text
-   ``shift`` + ``ctrl`` + ``z``    Insert a horizontal line
-   ``shift`` + ``ctrl`` + ``l``    Format as unordered list
-   ``shift`` + ``ctrl`` + ``k``    Format as ordered list
-   ``shift`` + ``ctrl`` + ``1``    Format as h1 heading
-   ``shift`` + ``ctrl`` + ``2``    Format as h2 heading
-   ``shift`` + ``ctrl`` + ``3``    Format as h3 heading
-   ``shift`` + ``ctrl`` + ``x``    Remove any hyperlink
-   ==============================  =============================================
-
+   ========================================   =================================================
+   Key / key combination                      Function
+   ========================================   =================================================
+   :kbd:`ctrl` :kbd:`u`                       Format text underlined
+   :kbd:`ctrl` :kbd:`b`                       Format text in **bold**
+   :kbd:`ctrl` :kbd:`i`                       Format text in *italics*
+   :kbd:`ctrl` :kbd:`s`                       Format text as  ̶s̶t̶r̶i̶k̶e̶t̶h̶r̶o̶u̶g̶h̶
+   :kbd:`ctrl` :kbd:`v`                       Paste text from clipboard
+   :kbd:`ctrl` :kbd:`shift` :kbd:`v`          Paste text from clipboard (as plain text)
+   :kbd:`shift` :kbd:`ctrl` :kbd:`f`          Remove formatting of text
+   :kbd:`shift` :kbd:`ctrl` :kbd:`y`          Remove formatting of the whole text
+   :kbd:`shift` :kbd:`ctrl` :kbd:`z`          Insert a horizontal line
+   :kbd:`shift` :kbd:`ctrl` :kbd:`l`          Format as unordered list
+   :kbd:`shift` :kbd:`ctrl` :kbd:`k`          Format as ordered list
+   :kbd:`shift` :kbd:`ctrl` :kbd:`1`          Format as h1 heading
+   :kbd:`shift` :kbd:`ctrl` :kbd:`2`          Format as h2 heading
+   :kbd:`shift` :kbd:`ctrl` :kbd:`3`          Format as h3 heading
+   :kbd:`shift` :kbd:`ctrl` :kbd:`x`          Remove any hyperlink
+   ========================================   =================================================
+
+.. tip:: If you are a Mac user, use :kbd:`cmd` instead of :kbd:`ctrl` and
+   :kbd:`ctrl` :kbd:`option` instead of :kbd:`ctrl` :kbd:`shift`.
```

Comment:
Just please reverse `:kbd:ctrl :kbd:shift` into `:kbd:shift :kbd:ctrl` at the very end, because these modifiers are always in the listed order, to make them different than the system ones (e.g. pasting as plain text, which is BTW the same on Mac, but `cmd+shift+v`, for sure).